seonoh/ArchitectureStudy/app/src/main/java/com/example/seonoh/seonohapp/CoinAdapter.kt

```diff
@@ -0,0 +1,29 @@
+package com.example.seonoh.seonohapp
+
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import androidx.recyclerview.widget.RecyclerView
+import com.example.seonoh.seonohapp.model.UseCoinModel
+
+class CoinAdapter : RecyclerView.Adapter<CoinItemViewHolder>() {
+
+    private val mData: ArrayList<UseCoinModel> by lazy { ArrayList<UseCoinModel>() }
+
+    override fun onCreateViewHolder(parent: ViewGroup, viewType: Int): CoinItemViewHolder {
+
+        val mView = LayoutInflater.from(parent.context).inflate(R.layout.coin_item, parent, false)
```

Comment:
전역변수가 아니기 때문에 좋지 않은 네이밍인 것 같습니다
```suggestion
        val view = LayoutInflater.from(parent.context).inflate(R.layout.coin_item, parent, false)
```

Reply:
네 수정했습니다!

Reply:
@seonoh 
반영 커밋 남겨주세요

Reply:
7896ebf